### PR TITLE
fix(Box): Fix imports and the variant

### PR DESCRIPTION
**Pull request by codecaaron**
### PR Checklist

- [x] Related to Abstract designs:
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@4.1.1-alpha.ae42c2.0<br/></pre>

File: packages/gamut-labs/src/experimental/Box/constants.ts [hunk 1]
```diff
@@ -1,11 +1,19 @@
 import { colors } from '@codecademy/gamut-styles';
 
 const { white, navy, yellow } = colors;
+export type BoxVariants = 'yellow' | 'navy' | 'white';
 
-export const BOX_VARIANTS = {
+export const BOX_VARIANTS: Record<
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/experimental/Box/constants.ts:6]: Any reason for a manual `Record` over `as const`?

You could do `export type BoxVariants = keyof typeof BOX_VARIANTS;` below.

**Review by JoshuaKGoldberg — APPROVED** [review]
The TS is a little more manual than I think we need but this otherwise looks good to me.

File: packages/gamut-labs/src/experimental/Box/constants.ts (hunk 1 above)
Comment (codecaaron) [reply packages/gamut-labs/src/experimental/Box/constants.ts:6]: I had to switch it now that shadow was optional on the type.  However I guess I could cast the restructuring to a "BoxVaiantShape" type instead.

What do you think?

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f68e610cf25471eeb564afd--gamut-preview.netlify.app>https://5f68e610cf25471eeb564afd--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f68e610cf25471eeb564afd>Deploy Logs</a></p>

File: packages/gamut-labs/src/experimental/Box/constants.ts (hunk 1 above)
Comment (JoshuaKGoldberg) [reply packages/gamut-labs/src/experimental/Box/constants.ts:6]: ehh, seems fine as is in that case 🤷 no hard feelings here. I see what you mean on the optional..

**Pull request merged**